src/app/signin/auth-ui.tsx

```diff
@@ -38,6 +39,16 @@ export default function AuthUI() {
         }}
         theme="light"
       />
+      <div className="text-sm flex flex-col text-muted-foreground">
+        <button type="button" onClick={() => seePassword()} class="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none dark:focus:ring-blue-800">See password</button>
+        Would you like to give it a try? Feel free to use our demo account 😉
+        <p>
+          email: <code className="text-sm ">tiffs_purview0r@icloud.com</code>
```

Comment:
Please update your main branch with the latest main in flowinance, this has already been changed, the demo is not accessible like this anymore.

Reply:
done

Reply:
I continue seeing it. Please check out: https://github.com/manuelalferez/flowinance/blob/main/src/app/signin/auth-ui.tsx#L22

Reply:
This part has been updated in previous features. You should update your branch with the last commits of main in order to fix this error